docs/quick-start-docker.md

```diff
@@ -85,14 +85,13 @@ Add a new data source with with these settings:
 * Name: `metrictank`
 * Default: `yes`
 * Type: `Graphite`
-* Url: `http://localhost:8080`
-* Access: `direct` (not `proxy`)
+* Url: `http://graphite`
+* Access: `proxy`
 
```

Comment:
graphite-web does not support CORS. so direct does not work.

users can still connect directly to the datasource on localhost:8080.